main.py

```diff
@@ -0,0 +1,10 @@
+# write your code here
+def copy_file(command: str) -> None:
+
+    first_name, second_name = command.split()[1], command.split()[2]

```

Comment:
It's better to create commands = command.split()
And use unpacking to get command and filenames